specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/examples/getSoftwareUpdates.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
minor - rename file to getsoftwareupdate.json

Reply:
Done